ovos_date_parser/dates_gl.py

```
@@ -0,0 +1,305 @@
+        def substitucion(match):
+            unidades_tempo[unit_en] += float(match.group("value"))
+            return ''
+
+        text = re.sub(patron_unidade, substitucion, text)
+
+    for (unit_en, unit_gl) in unidades_non_estandar.items():
+        patron_unidade = patron.format(unit=unit_gl[:-1])  # eliminar 's' da unidade
+
+        def substitucion_non_estandar(match):
+            val = float(match.group("value"))
+            if unit_en == "months":
```

Ruff (0.8.2)

Function definition does not bind loop variable `unit_en`

(B023)